Assets/Scripts/SuddenDeath.cs

```diff
@@ -82,7 +83,10 @@ private IEnumerator Run()
             if (t >= 1)
             {
                 timer = 0;
-                ActivateDoubling();
+                if (NoMultiply)
+                    SpawnDeathBall();
+                else
+                ActivateDoubling(); 
```

Comment:
Shouldn't this also have an indent? Not familiar with C# so not sure here.

Reply:
Also not familiar with C#, lol. VisualStudio took the indent away when I put it in, so I figured it was fine.
Testing on my stream indicates that the code should function, though.